Code/Core/Math/Random.cpp

```diff
@@ -24,4 +26,15 @@ uint32_t Random::GetRand()
     return( (uint32_t)( m_Seed / ( ( CORE_RAND_MAX + 1 ) * 2 ) ) % ( CORE_RAND_MAX + 1 ) );
 }
 
+// GetRandString
+//------------------------------------------------------------------------------
+uint32_t Random::GetRandString( AString& str )
```

Comment:
Minor: FASTBuild's style includes a space before the &.

I can easily fix these during integration, but it'll be easier if that's not needed.

Reply:
Done here and in a few more places.